install_config/install/deploy_router.adoc

```diff
@@ -875,6 +875,63 @@ endif::[]
 
 Save this content to a file for use as the basis of your customized template.
 
+[[using-configmap]]
+
+=== Using ConfigMap
+
+A link:../../dev_guide/configmaps.html[configMap] permits you to customize
+the router instance without rebuilding the router image. The
+*_haproxy-config.template_*, *_reload-haproxy_* and other scripts can be modified
+as well as creating and modifying router environment variables.
+
+The following example shows how to use a custom *_haproxy-config.template_* in
+your router.
+
+First get a copy of the *_haproxy-config.template_* that you want to modify
+as described above. Modify the *_haproxy-config.template_* as desired.
+
+Create a configmap:
+====
+----
+$ oc create configmap customrouter --from-file=haproxy-config.template
+----
+====
+The configMap contains a copy of your modified *_haproxy-config.template_* file.
+The router dc is next modified to mount the configMap as a file and the `TEMPLATE_FILE`
+environment variable is modified to point to it.
+
+Now edit the router dc to mention the configMap in all the needed places.
```

Comment:
Can we do this with 'oc env' and 'oc volume'?

I think that is easier for people to do.